### PR TITLE
Setup python system tests

### PR DESCRIPTION
**Pull request by andrewkroh**
- Setup python system tests for project.
- Adding Vagrant file to help with development/testing.

### CODE REVIEW

File: scripts/Makefile [hunk 1]
```diff
@@ -107,7 +106,8 @@ system-tests: winlogbeat.test prepare-tests system-tests-setup
 	# Writes atomic mode on top of file
 	echo 'mode: atomic' > ${COVERAGE_DIR}/system.cov
 	# Collects all system coverage files and skips top line with mode
-	tail -q -n +2 ./build/system-tests/run/**/*.cov >> ${COVERAGE_DIR}/system.cov
+	# TODO: Uncomment when there are system-tests that produce coverage reports.
```
Comment (tsg) [inline scripts/Makefile]: Doesn't this PR add system tests? I'm thinking this comment can be reverted already.

**Comment by tsg** [comment]
LGTM, see just the question above.

File: scripts/Makefile (hunk 1 above)
Comment (andrewkroh) [inline scripts/Makefile]: @tsg No coverage reports are produced when I run these two tests. I'm not really sure why.

Comment (tsg) [inline scripts/Makefile]: I see, ok.

**Pull request merged**